src/v2/Apps/Settings/SettingsApp.tsx

```diff
@@ -13,6 +13,7 @@ const TABS = [
   { name: "Edit Settings", url: "/settings/edit-settings" },
   { name: "Collector Profile", url: "/settings/edit-profile" },
   { name: "Saves & Follows", url: "/settings/saves" },
+  { name: "Your Alerts", url: "/user/alerts" },
```

Comment:
We should also change the route to `/settings/alerts` and add a redirect for the old route here: https://github.com/artsy/force/blob/a5a3defb3ba8ae6f382ffc5c3ebed85621d3f353/src/lib/middleware/hardcodedRedirects.ts#L77-L83

Reply:
Got it! Thanks for pointing this out.